syndeo-template/src/main/java/com/google/cloud/syndeo/SyndeoTemplate.java

```diff
@@ -68,6 +69,29 @@ public interface Options extends PipelineOptions, DataflowPipelineOptions {
     void setJsonSpecPayload(String jsonSpecPayload);
   }
 
+  public static final Set<String> SUPPORTED_URNS =
+      Set.of(
+          // Old names:
+          "bigquery:read",
+          "kafka:read",
+          "bigtable:write",
+          "schemaIO:bigquery:read",
+          "schemaIO:bigquery:write",
+
+          // New names:
+          "beam:schematransform:org.apache.beam:sql_transform:v1",
```

Comment:
Should we be adding all of these right now? Currently we only support Kafka Read, BigQuery Read + Write, Bigtable Write, and Pubsub Read. I'm not sure if we should be adding the unsupported ones as well, but I would defer to another opinion